actions/common.go

```diff
@@ -71,6 +71,17 @@ var (
 	}
 	defaultStagingPath = getFullPath("staging")
 
+	// RouterChart represents the router chart and its files needing updating
+	// for a release
+	RouterChart = helmChart{
+		Name:     "router-dev",
+		Template: generateParamsRouterTpl,
+		Files: []string{
+			"README.md",
+			"Chart.yaml",
+		},
+	}
```

Comment:
is it worth putting this into a slice, so we can generate more individual charts?